### PR TITLE
Update network-exchange-clc-endpoint-guide.md

### PR DESCRIPTION
**Pull request by marcopao**
Added bullet item for migrating existing CLC accounts to dedicated VR, deleted SLA reference, added links for Availability Matrix in two spots, slight wording change in next to last bullet and added bullet item and link to NetX Architecture Guide.

### CODE REVIEW

**Comment by jim-phillips** [comment]
@marcopao Please resolve conflicts.

**Comment by marcopao** [comment]
Resolved conflicts per request.

File: Network/network-exchange-clc-endpoint-guide.md [hunk 1]
```diff
@@ -13,7 +13,9 @@
...
 * For any existing CLC accounts that wish to use Network Exchange, it is necessary to migrate them to a Dedicated VR.  Having a dedicated virtual routing table allows each customer to have their own set of IPs’ from a cloud connectivity standpoint.  A Dedicated VR is also required for automation to work when creating connectivity to the cloud.  
 To create a dedicated virtual routing table (Dedicated VR), the VR will need to be created and all existing IP blocks the customer is using will need to converge to the new VR.  To initiate this process, create a ticket with the CenturyLink Cloud help desk.  
-* The desired CenturyLink Cloud must be supported within Network Exchange. See the *Network Exchange Availability Matrix and Configuration Guide* for supported data centers.
+* The desired CenturyLink Cloud must be supported within Network Exchange. See the [Network Exchange Availability Matrix and Configuration Guide](https://www.ctl.io/knowledge-base/network/network-exchange-connectivity-matrix-configuration-guide/)
```
Comment (jim-phillips) [inline Network/network-exchange-clc-endpoint-guide.md]: Links to other KB articles need to reference the file system.
`[./network-exchange-connectivity-matrix-configuration-guide.md]`

File: Network/network-exchange-clc-endpoint-guide.md [hunk 2]
```diff
@@ -25,6 +27,8 @@ To create a dedicated virtual routing table (Dedicated VR), the VR will need to
 ### Notes
 
 * The Network Exchange fabric leverages the economics of shared networking while logically isolating network traffic between Exchanges, even within the same End User account. Dedicated access to CenturyLink Cloud is not supported with Network Exchange.
-* Currently, an End User may only add the CenturyLink Cloud endpoint(s) present in the same metropolitan area as the serving instance of Network Exchange. Where more than one CLC endpoint is served, all may be included in a given Exchange. See the *Network Exchange Availability Matrix and Configuration Guide* for more information.
+* Currently, an End User may only add the CenturyLink Cloud endpoint(s) present in the same metropolitan area as the serving instance of Network Exchange. Where more than one CLC endpoint is served, all may be included in a given Exchange. See the [Network Exchange Availability Matrix and Configuration Guide](https://www.ctl.io/knowledge-base/network/network-exchange-connectivity-matrix-configuration-guide/) for available endpoints per metro area.
```
Comment (jim-phillips) [inline Network/network-exchange-clc-endpoint-guide.md]: Links to other KB articles need to reference the file system.
`[./network-exchange-connectivity-matrix-configuration-guide.md]`

File: Network/network-exchange-clc-endpoint-guide.md [hunk 3]
```diff
@@ -25,6 +27,8 @@ To create a dedicated virtual routing table (Dedicated VR), the VR will need to
...
 * A CLC service ticket is not required to add or delete a CenturyLink Cloud endpoint.
-* For additional information regarding the Network Exchange architecture and use cases, please consult the *Network Exchange Architecture and Technical Guide*.
+* For additional information regarding the Network Exchange architecture and use cases, please consult the [Network Exchange Architecture and Technical Guide](https://cloudhelp.ctl.io/knowledge-base/network-exchange-technical-guide/)
```
Comment (jim-phillips) [inline Network/network-exchange-clc-endpoint-guide.md]: I don't think we should be linking to the internal cloud help site in a Public KB.

**Comment by jim-phillips** [comment]
@marcopao last chance to fix this.  I will be closing due to inactivity on 4/11

**Pull request closed without merging**